istio/tests/common.py

```diff
@@ -246,6 +250,10 @@
     'istio_request_duration_seconds': 'request.duration',
     'istio_request_bytes': 'request.size',
     'istio_response_bytes': 'response.size',
+    'istio_tcp_connections_closed_total': 'tcp.connections_closed.total',
+    'istio_tcp_connections_opened_total': 'tcp.connections_opened.total',
+    'istio_tcp_received_bytes_total': 'tcp.received_bytes.total',
+    'istio_tcp_sent_bytes_total': 'tcp.send_bytes.total',
```

Comment:
It seems these are not present in the fixtures for 1.4. Is it an oversight, or are these metrics actually not provided before 1.5? Seems like this is what is breaking CI currently.

Reply:
These metrics are only available from `1.4+` I believe. However, we aren't testing `1.4` fixtures. I've updated the test metrics list and properly labelled the fixtures (it was from a pre 1.0.0 release in 2018).